shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-core/src/test/java/org/apache/shardingsphere/proxy/frontend/mysql/MockMySQLProtocolFrontendEngine.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.proxy.frontend.mysql;
+
+import org.apache.shardingsphere.db.protocol.codec.DatabasePacketCodecEngine;
+import org.apache.shardingsphere.proxy.backend.communication.jdbc.connection.BackendConnection;
+import org.apache.shardingsphere.proxy.frontend.context.FrontendContext;
+import org.apache.shardingsphere.proxy.frontend.engine.AuthenticationEngine;
+import org.apache.shardingsphere.proxy.frontend.engine.CommandExecuteEngine;
+import org.apache.shardingsphere.proxy.frontend.spi.DatabaseProtocolFrontendEngine;
+
+public final class MockMySQLProtocolFrontendEngine implements DatabaseProtocolFrontendEngine {
```

Comment:
It is better to rename `MockMySQLProtocolFrontendEngine` to `MockProtocolFrontendEngine`, because MySQL is just a type of implementation.

Reply:
OK, you are right, I will change this.

Reply:
CI failed due to:
[ERROR] Tests run: 4, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.98 s <<< FAILURE! - in org.apache.shardingsphere.proxy.governance.schema.ProxyGovernanceSchemaContextsTest
[ERROR] unnecessary Mockito stubbings(org.apache.shardingsphere.proxy.governance.schema.ProxyGovernanceSchemaContextsTest)  Time elapsed: 0.034 s  <<< ERROR!
org.mockito.exceptions.misusing.UnnecessaryStubbingException: 

Unnecessary stubbings detected in test class: ProxyGovernanceSchemaContextsTest
Clean & maintainable test code requires zero unnecessary code.
Following stubbings are unnecessary (click to navigate to relevant line of code):
  1. -> at org.apache.shardingsphere.proxy.governance.schema.ProxyGovernanceSchemaContextsTest.getProxyGovernanceSchemaContexts(ProxyGovernanceSchemaContextsTest.java:83)
Please remove unnecessary stubbings or use 'silent' option. More info: javadoc for UnnecessaryStubbingException class.